### DISCUSSION TITLE
Error while installing MOOSE on HPC

### DISCUSSION THREAD

**Discussion by yuezhiying6** [idaholab/moose#27209] [Q&A Getting Started]
Dear all,

I am installing MOOSE on HPC  according to the instructions in the official documentation. However, some errors occurred when compiling the superlu_dist. Following is the configure.log file after I run the command in the terminal ./moose/scripts/update_and_rebuild_petsc.sh. Any suggestion would be appreciated. 
[configure.log](https://github.com/idaholab/moose/files/14786142/configure.log)

Zhiying 
Best regards

**Comment by GiudGiud** [comment]
Hello

you are using the intel compilers which are not supported by moose, and looking at the log also not by superlu 

you will need to clean your environment and have only supported compilers (gnu clang) available

you can use ‘env’ to check or the diagnostics script in moose/scripts

Guillaume
